### PR TITLE
Supports FRR-VRRP configuration

### PR DESCRIPTION
**Pull request by philo-micas**
hld:https://github.com/sonic-net/SONiC/pull/1446

### CODE REVIEW

File: common/schema.h [hunk 1]
```diff
@@ -410,7 +410,8 @@ namespace swss {
 #define CFG_MCLAG_UNIQUE_IP_TABLE_NAME              "MCLAG_UNIQUE_IP"
 
 #define CFG_PORT_STORM_CONTROL_TABLE_NAME           "PORT_STORM_CONTROL"
-
+#define CFG_VRRP_TABLE_NAME                         "VRRP"
+#define CFG_VRRP6_TABLE_NAME                        "VRRP6"
```
Comment (nmoray) [inline common/schema.h:414]: Need similar table entry for APPL_DB too.

`#define APP_VRRP_TABLE_NAME  "VRRP_TABLE"`

Comment (philo-micas) [reply common/schema.h:427]: sure, has done.

**Review by nmoray — APPROVED** [review]

**Comment by vvbrcm** [comment]
Looks fine, please go ahead with merge.

**Review by vvbrcm — APPROVED** [review]

**Comment by adyeung** [comment]
@qiluo-msft please help merge this, reviewed and approved by BRCM and Aviz

**Review by qiluo-msft — DISMISSED** [review]

**Review by yuezhoujk — APPROVED** [review]

**Comment by wen587** [comment]
Hi @philo-micas , could you help add yang model for this table

**Comment by philo-micas** [comment]
Hi @wen587 , there is already a PR for yang model. 
Here's the link: https://github.com/sonic-net/sonic-buildimage/pull/20383

**Pull request merged**